lib/src/span.dart

```diff
@@ -48,10 +48,11 @@ abstract class SourceSpan implements Comparable<SourceSpan> {
   /// Formats [message] in a human-friendly way associated with this span.
   ///
   /// [color] may either be a [String], a [bool], or `null`. If it's a string,
-  /// it indicates an ANSII terminal color escape that should be used to
-  /// highlight the span's text. If it's `true`, it indicates that the text
-  /// should be highlighted using the default color. If it's `false` or `null`,
-  /// it indicates that the text shouldn't be highlighted.
+  /// it indicates an ANSI terminal color escape that should be used to
+  /// highlight the span's text (e.g. `"\u001b[31m"` will color red). If it's
```

Comment:
Nit: [avoid "e.g."](https://www.dartlang.org/guides/language/effective-dart/documentation#avoid-abbreviations-and-acronyms-unless-they-are-obvious).

Reply:
Done.